pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprsplit.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  meson,
+  fetchFromGitHub,
+  hyprland,
+  ninja,
+  mkHyprlandPlugin,
+}:
+mkHyprlandPlugin hyprland rec {
+  pluginName = "hyprsplit";
+  version = "0.43.0";
+
+  src = fetchFromGitHub {
+    owner = "shezdy";
+    repo = "${pluginName}";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-r533kNIyfgPi/q8ddIYyDK1Pmupt/F3ncHuFo3zjDkU=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+
+  dontStrip = true;
```

Comment:
why no stripping? Stripping would reduce the closure size.

Reply:
I thought that because of their nature of plugins it was mandatory to not strip the output lib but it seems I was wrong as it works even when doing it. so corrected.